### PR TITLE
[SOL-1835] Add guideline link to catalog query string.

### PR DESCRIPTION
**Pull request by vkaracic**
screenshot from 2016-06-27 18-08-57

---

https://openedx.atlassian.net/browse/SOL-1835
Bullet points 1, 2 and 3 have been addressed in previous work:

---

screenshot from 2016-06-27 18-11-38

---

screenshot from 2016-06-27 18-11-32

Reviewers:
- [x] @srpearce 
- [x] Roderick Morales

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @vkaracic! It looks like you're a member of a company that does contract work for edX. If you're doing this work as part of a paid contract with edX, you should talk to edX about who will review this pull request. If this work is not part of a paid contract with edX, then you should ensure that there is an OSPR issue to track this work in [JIRA](https://openedx.atlassian.net), so that we don't lose track of your pull request.

To automatically create an OSPR issue for this pull request, just visit this link: https://openedx-webhooks.herokuapp.com/github/process_pr?repo=edx%2Fecommerce&number=829

File: ecommerce/static/templates/coupon_form.html [hunk 1]
```diff
@@ -181,7 +181,7 @@
             <p class="help-block"></p>
         </div>
         <div class="form-group catalog-query">
-            <label for="catalog-query"><%= gettext('Valid for:') %> *</label>
+            <label for="catalog-query"><%= gettext('Query string:') %> * (<a href="https://stage-edx-discovery.edx.org/" class="normal-font-weight">guideline</a>)</label>
```
Comment (srpearce) [inline ecommerce/static/templates/coupon_form.html]: guideline --> query hints

I would also make the parentheses around "query hints" part of the link - it looks a little funny to have the parentheses in bold dark gray and the link text in non-bold blue.

Comment (vkaracic) [inline ecommerce/static/templates/coupon_form.html]: Here's an updated version:
screenshot from 2016-06-28 09-58-23

**Comment by roderickmorales** [comment]
@srpearce I don't want to get too picky but is "hints" the right word here? Isn't more like "query syntax documentation"... "hint" sounds like a clue instead of a guide.

**Comment by srpearce** [comment]
How about "query guidelines"? (I didn't want to have just "guidelines" because it didn't specify what the guidelines were for, and I was thinking in terms of screen readers.)

**Comment by vkaracic** [comment]
`query guidelines` it is!

**Comment by roderickmorales** [comment]
👍

**Pull request merged**